src/pepr/operator/reconcilers/reconciler.ts

```diff
@@ -1,12 +1,13 @@
-import { K8s, Log } from "pepr";
```

Comment:
Small nit: I like the foldering of reconcilers, but might be good to rename this one to `package-reonciler.ts` (and same with the spec file).